File: pyg-user-web/src/main/webapp/js/controller/loginController.js
//定义控制器
//依赖注入service
app.controller("loginController", function($scope,loginService) {
	
	//获取用户登录成功用户名
	$scope.loadLoginName = function() {
		// 发送请求
		loginService.loadLoginName().success(function(data) {
			$scope.loginName = data.loginName;
		})
	};	

	

})

File: pyg-user-web/src/main/java/com/pyg/user/controller/LoginController.java
package com.pyg.user.controller;

import java.util.HashMap;
import java.util.Map;

import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/login")
public class LoginController {
	
	/**
	 * 需求：获取单点登录用户名
	 */
	@RequestMapping("loadLoginName")
	public Map<String, String> loadLoginName(){
		String username = SecurityContextHolder.getContext().getAuthentication().getName();
		//创建map对象
		Map<String, String> maps = new HashMap<>();
		maps.put("loginName", username);
		
		return maps;
	}

}
